web/src/features/charts/bar-breakdown/utils.ts

```diff
@@ -30,21 +30,19 @@ export function getProductionCo2Intensity(
   }
 
   const storage = zoneData.storage?.[generationMode as ElectricityStorageKeyType];
-  // TODO: Find out how this worked before if the data is never available
-  const storageCo2Intensity = zoneData.storageCo2Intensities?.[generationMode];
```

Comment:
There is no storageCo2Intensities key in the zoneData.

Reply:
Yes, agree - but there also shouldn't be (see https://github.com/electricitymaps/electricitymaps-contrib/issues/3668)